src/components/ContentPlaceholder/ContentPlaceholder.scss

```diff
@@ -0,0 +1,35 @@
+@keyframes shimmer {
+    0% {
+        background-position: 100% 0;
+    }
+
+    100% {
+        background-position: -100% 0;
+    }
+}
+
+.ContentPlaceholder {
+    animation:{
+        duration: 1s;
```

Comment:
since it is set in the style element, it is redundant here. can it be removed? Or you think it is makes it more readable?

Reply:
It's not set as style attribute by default (this is the default). Only if the prop is actually used to apply a custom duration, it will be declared inline (which takes precedence over the stylesheet declaration).

Reply:
yes. that what I meant. have it inline only. But have it there always (with default prop set to 1). Then the default and the custom works in the same way, and is easier to understand.